Gemfile.lock

```diff
@@ -121,36 +122,35 @@ GEM
       activesupport (>= 3.0.0)
       uniform_notifier (~> 1.11)
     byebug (11.1.3)
-    capistrano (3.14.0)
+    capistrano (3.14.1)
       airbrussh (>= 1.0.0)
       i18n
       rake (>= 10.0.0)
       sshkit (>= 1.9.0)
-    capistrano-bundler (1.6.0)
+    capistrano-bundler (2.0.1)
       capistrano (~> 3.1)
-    capistrano-rails (1.5.0)
+    capistrano-rails (1.6.1)
       capistrano (~> 3.1)
-      capistrano-bundler (~> 1.1)
+      capistrano-bundler (>= 1.1, < 3)
     capistrano-rails-console (2.3.0)
       capistrano (>= 3.5.0, < 4.0.0)
       sshkit-interactive (~> 0.3.0)
     case_transform (0.2)
       activesupport
-    code_analyzer (0.5.1)
+    code_analyzer (0.5.2)
       sexp_processor
     codeclimate-engine-rb (0.4.1)
       virtus (~> 1.0)
     coderay (1.1.3)
     coercible (1.0.0)
       descendants_tracker (~> 0.0.1)
-    concurrent-ruby (1.1.6)
-    crack (0.4.3)
-      safe_yaml (~> 1.0.0)
+    concurrent-ruby (1.1.7)
+    crack (0.4.4)
```

Comment:
### [crack](http://github.com/jnunemaker/crack)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 0.4.3 → 0.4.4

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **5** commits. See the full changes on [the compare page](https://github.com/jnunemaker/crack/compare/v0.4.3...v0.4.4).

These are the individual commits:
- (a7189fe) [remove safe_yaml](https://github.com/jnunemaker/crack/commit/a7189fef728ce522cb03f62b18848cfc50058245)
- (261962a) [Exclude tests (and fixtures) from bundled gem](https://github.com/jnunemaker/crack/commit/261962ae56f0c4216d0be6b8c65037ec5c853a70)
- (1183581) [Merge pull request #61 from greysteil/dont-bundle-tests](https://github.com/jnunemaker/crack/commit/1183581ded7d7ee6b427778eef430d3d10000e04)
- (4ecf9cf) [Merge pull request #59 from bf4/master](https://github.com/jnunemaker/crack/commit/4ecf9cf5f721dea61b8fa86280180142ead35312)
- (288371d) [Bump version to 0.4.4; update history](https://github.com/jnunemaker/crack/commit/288371d0444ba64bde1b082f340fcc67eeb9e64e)


</details>